File: HabitTracker/Contracts/Habit.Tracker.Contracts/Dtos/User/Login/LoginRequestDto.cs
using System.ComponentModel;

namespace Habit.Tracker.Contracts.Dtos.User.Login;

public class LoginRequestDto
{
    [DefaultValue("ahmet@mail")]
    public string Email { get; set; }
    [DefaultValue("123Ahmet")]
    public string Password { get; set; }

}

File: HabitTrackerFlutter/lib/models/Login/login_request_dto.dart
class LoginRequestDto {
  String email;
  String password;

  LoginRequestDto({
    required this.email,
    required this.password,
  });

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = {
      'Email': email,
      'Password': password,
    };
    return data;
  }
}